lib/s3.rb

```diff
@@ -0,0 +1,21 @@
+class S3
```

Comment:
https://stackoverflow.com/questions/37799296/ruby-what-does-the-comment-frozen-string-literal-true-do 🤷 

```suggestion
# frozen_string_literal: true

class S3
```